stout/flags.cc

```diff
@@ -101,6 +190,18 @@ void Parser::Parse(int* argc, const char*** argv) {
 
     // Skip anything that doesn't look like a flag.
     if (arg.find("--") != 0) {
+      // It could be probably subcommand.
+      if (SetCurrentMessageForSubcommand(arg, message_)) {
+        if (subcommands_.count(arg) > 0) {
+          std::cerr << "Encountered duplicate subcommand '"
+                    << arg << "'." << std::endl;
+          exit(1);
+        } else {
+          subcommands_.insert(arg);
+        }
+        AddAllOrExit(*cur_subcommand_message_);
+        continue;
+      }
```

Comment:
Don't want to add to `args` if we parse as a subcommand!
```suggestion
      } else {
```